cmd/deploy/deploy.go

```diff
@@ -212,48 +213,42 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 		}
 
 		oktetoLog.Information("Checking to build services at manifest")
-		if deployOptions.Manifest.Build != nil {
-
-			for service, mOptions := range deployOptions.Manifest.Build {
-				buildOptions := build.BuildOptions{OutputMode: oktetoLog.GetOutputFormat()}
-				if mOptions.Name == "" {
-					mOptions.Name = deployOptions.Name
+		if deployOptions.Build {
+			// force build of all services
```

Comment:
Could we encapsulate this into a function?

Reply:
yes we can!